src/pages/guides/rust/metaplex-rust-sdks.md

```diff
@@ -0,0 +1,197 @@
+Lets take the `CreateV1` instruction from Core as an example (this applies to all other instructions from this Crate and all other Metaplex crates too).
+
+If we look through the instructions in the  we can see we have a number of instructions available to us.
+
+### Builder
+
+Builder instructions are designed to be used via
+
+The one we are interested in here is the `CreateV1Builder`.
+
+To initialize the builder we can call `new`.
+
+```rust
+CreateV1Builder::new();
+```
+
+From this point we can `ctrl + click` (pc) or `cmd + click` (mac) into the `new` function generated from the `Builder::` which positions us at the `pub fn new()` for the builder. If you scroll up slightly you'll then see the `pub struct` for the `CreateV1Builder` as outlined below.
+
+```rust
+pub struct CreateV1Builder {
+    asset: Option<solana_program::pubkey::Pubkey>,
+    collection: Option<solana_program::pubkey::Pubkey>,
+    authority: Option<solana_program::pubkey::Pubkey>,
+    payer: Option<solana_program::pubkey::Pubkey>,
+    owner: Option<solana_program::pubkey::Pubkey>,
+    update_authority: Option<solana_program::pubkey::Pubkey>,
+    system_program: Option<solana_program::pubkey::Pubkey>,
+    log_wrapper: Option<solana_program::pubkey::Pubkey>,
+    data_state: Option<DataState>,
+    name: Option<String>,
+    uri: Option<String>,
+    plugins: Option<Vec<PluginAuthorityPair>>,
+    __remaining_accounts: Vec<solana_program::instruction::AccountMeta>,
+}
+
+```
+
+These are your arguments of publickeys and data that will need to be passed into the builder. Some accounts may also be optional and default to others, this can vary from instruction to instruction. If you click through to the `new()` function again and scroll down this time you'll see the individual functions with additional comments. In the below case you can see that the owner will default to payer, so we don't need to pass in owner if in this case if the payer is also going to be the owner of the Asset.
+
+```rust
+/// `[optional account]`
+    /// The owner of the new asset. Defaults to the authority if not present.
+    #[inline(always)]
+    pub fn owner(&mut self, owner: Option<solana_program::pubkey::Pubkey>) -> &mut Self {
+        self.owner = owner;
+        self
+    }
+```
+
+Here is an example using the `CreateV1Builder` that returns an instruction using `.instruction()` to close out the Builder.
+
+```rust
+let create_asset_ix = CreateV1Builder::new()
+        .asset(asset.pubkey())
+        .collection(collection.pubkey())
+        .payer(payer.pubkey())
+        .name("My Nft".into())
+        .uri("https://example.com/my-nft.json".into())
+.       .instruction();
+```
+
+Now that we have our instruction ready we need to create a normal Solana transaction to send to our RPC. This includes a blockhash andxx§ signers.
+
+### Full Builder Example
+
+This is a full example of creating a instruction using a Metaplex `Builder` function and sending that transction off to the chain.
```

Comment:
Great examples.

Idea: should we also create a collection somewhere in the guide and then create an asset in the collection?  I think that is at least 90% use case for creating assets?

Reply:
Should this be it's own guide "creating a collection"?. I think we discuss this as a group as I feel just creating a barebones nft is possibly the first guide.